test/DynamicHelpTest.cs

```diff
@@ -0,0 +1,196 @@
+                        NextLine,
+                        "Input: True (ByPropertyName, ByValue), WildCard: false")),
+                _.Enter,
+                _.Enter,
+                    CheckThat(() => AssertScreenIs(1,
+                        TokenClassification.Command, "Get-Date",
+                        TokenClassification.None, " ",
+                        TokenClassification.Parameter, "-Date"))
+                ));
+        }
+
+        [SkippableFact]
+        public void DynHelp_GetParameterHelpMultiLine_And_Clear()
+        {
+            TestSetup(KeyMode.Cmd);
+            string emptyLine = new string(' ', _console.BufferWidth);
+
+            Test("Get-MultiLineHelp -OneAndHalf", Keys(
+                "Get-MultiLineHelp -OneAndHalf", _.Alt_h,
+                    CheckThat(() => AssertScreenIs(9,
+                        TokenClassification.Command, "Get-MultiLineHelp",
+                        TokenClassification.None, " ",
+                        TokenClassification.Parameter, "-OneAndHalf",
+                        NextLine,
+                        emptyLine,
+                        TokenClassification.None, $"-Date <name>",
+                        NextLine,
+                        emptyLine,
+                        TokenClassification.None, "DESC: Some very long description that is over the buffer width of ",
+                        TokenClassification.None, "60 characters but shorter than 120.",
+                        NextLine,
+                        TokenClassification.None, "Required: false, Position: 0, Default Value: None, Pipeline ",
+                        NextLine,
+                        "Input: True (ByPropertyName, ByValue), WildCard: false")),
+                _.Enter,
+                _.Enter,
+                    CheckThat(() => AssertScreenIs(1,
+                        TokenClassification.Command, "Get-MultiLineHelp",
+                        TokenClassification.None, " ",
+                        TokenClassification.Parameter, "-OneAndHalf"))
+                ));
+        }
+
+        [SkippableFact]
+        public void DynHelp_GetParameterHelpErrorMessage()
+        {
+            TestSetup(KeyMode.Cmd);
+            string emptyLine = new string(' ', _console.BufferWidth);
+
+            Test("Get-FakeHelp -Fake", Keys(
+                "Get-FakeHelp -Fake", _.Alt_h,
+                    CheckThat(() => AssertScreenIs(4,
+                    TokenClassification.Command, "Get-FakeHelp",
+                    TokenClassification.None, " ",
+                    TokenClassification.Parameter, "-Fake",
+                    NextLine,
+                    emptyLine,
+                    TokenClassification.None, "No help content available. Please use Update-Help to downloa",
+                    NextLine,
+                    "d the latest help content.")),
+                _.Enter,
```

Comment:
Similar here, maybe choose a different key to clear.

Reply:
Chose different keys for various tests